src/provider/bn254_grumpkin.rs

```diff
@@ -0,0 +1,367 @@
+//! This module implements the Nova traits for bn256::Point, bn256::Scalar, grumpkin::Point, grumpkin::Scalar.
+use crate::{
+  provider::{
+    keccak::Keccak256Transcript,
+    pedersen::CommitmentEngine,
+    poseidon::{PoseidonRO, PoseidonROCircuit},
+  },
+  traits::{CompressedGroup, Group, PrimeFieldExt, TranscriptReprTrait},
+};
+use digest::{ExtendableOutput, Input};
+use ff::{FromUniformBytes, PrimeField};
+use num_bigint::BigInt;
+use num_traits::Num;
+use pasta_curves::{
+  self,
+  arithmetic::{CurveAffine, CurveExt},
+  group::{cofactor::CofactorCurveAffine, Curve, Group as AnotherGroup, GroupEncoding},
+};
+use rayon::prelude::*;
+use sha3::Shake256;
+use std::io::Read;
+
+use halo2curves::bn256::{
+  G1Affine as Bn256Affine, G1Compressed as Bn256Compressed, G1 as Bn256Point,
+};
+use halo2curves::grumpkin::{
+  G1Affine as GrumpkinAffine, G1Compressed as GrumpkinCompressed, G1 as GrumpkinPoint,
+};
+
+/// Re-exports that give access to the standard aliases used in the code base, for bn256
+pub mod bn256 {
```

Comment:
@huitseeker maybe this file should be renamed to `bn256_grumpkin` to be consistent with halo2curves

Reply:
The types that this is exporting are all defined there: https://github.com/privacy-scaling-explorations/halo2curves/blob/main/src/bn256/mod.rs
Do you mean the module at the top of the file or the line you're commenting on?